Scripts/ParallaxDepthArtifacts.bv.lua

```diff
@@ -0,0 +1,65 @@
+----------------------------------------------------------------------------------------------------
+--
+-- Copyright (c) Contributors to the Open 3D Engine Project.
+-- For complete copyright and license terms please see the LICENSE at the root of this distribution.
+--
+-- SPDX-License-Identifier: Apache-2.0 OR MIT
+--
+--
+--
+----------------------------------------------------------------------------------------------------
+
+g_screenshotOutputFolder = ResolvePath('@user@/Scripts/Screenshots/ParallaxDepthArtifacts/')
+Print('Saving screenshots to ' .. NormalizePath(g_screenshotOutputFolder))
+
+OpenSample('Features/Parallax')
+ResizeViewport(512, 512)
+
+-- Several times there have bugs that appeared, related to inconsistent depth calculations causing unwanted clipping of pixels on parallax surfaces.
```

Comment:
nit: have bugs->have been bugs

Reply:
Or even better
"Several times there have been bugs that appeared," -> "There have been several bugs related..."